docs/source/external_tutorials/tutorial_cellpose_segmentation.ipynb

```diff
@@ -0,0 +1,400 @@
+      "text/plain": [
+       "<Figure size 720x1440 with 3 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Load the image and visualize its channels. We will use the first channel for segmentation.\n",
+    "\n",
+    "img = sq.datasets.visium_fluo_image_crop()\n",
+    "crop = img.crop_corner(1000, 1000, size=1000)\n",
+    "\n",
+    "fig, axes = plt.subplots(1, 3, figsize=(10, 20))\n",
+    "for i, ax in enumerate(axes):\n",
+    "    crop.show(\"image\", channel=i, ax=ax)"
```

Comment:
`img.show` should now have a `channelwise` argument that you can use for this plot. No need to define a separate figure and axes, just `crop.show('image', channelwise=True)`. Its possible that this is only on the `dev` branch for now, but we will merge all of that very soon.